c2corg_ui/static/js/maincontroller.js

```diff
@@ -92,3 +97,9 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 };
+/**
+ * @export
+ */
+app.MainController.prototype.updateMapResize = function() {
+  this.scope_ .$root.$emit('updateMapResize');
```

Comment:
OK, than in the advanced search controller? I'm not very found with having very specific pieces of code in the main controller.